.ci/scripts/all.sh

```diff
@@ -6,6 +6,21 @@ test -z "$srcdir" && srcdir=.
 # shellcheck disable=SC1090
 . "${srcdir}/common.sh"
 
-DEFAULT_COMPOSE_ARGS="${ELASTIC_STACK_VERSION} ${BUILD_OPTS} --no-apm-server-dashboards --no-apm-server-self-instrument --with-agent-rumjs --with-agent-dotnet --with-agent-go-net-http --with-agent-nodejs-express --with-agent-ruby-rails --with-agent-java-spring --with-agent-python-django --with-agent-python-flask --force-build --no-xpack-secure"
+DEFAULT_COMPOSE_ARGS="${ELASTIC_STACK_VERSION} ${BUILD_OPTS}\
+  --no-apm-server-dashboards \
+  --no-apm-server-self-instrument \
+  --with-agent-rumjs \
+  --with-agent-dotnet \
+  --with-agent-go-net-http \
+  --with-agent-nodejs-express \
+  --with-agent-ruby-rails \
+  --with-agent-java-spring \
+  --with-agent-python-django \
+  --with-agent-python-flask \
```

Comment:
Small refactor for follow-ups: could we define an array of agents and loop them adding a `--with-agent-$agent`?

Reply:
IMHO this would add an unnecessary complexity so add 8 values to a variable.

Reply:
SGTM 👍